modules/setup/config/__init__.py

```diff
@@ -154,11 +162,18 @@ def config_from_to(exchange, backtesting_from: int, backtesting_to: int, backtes
         backtesting_to_parsed = last_closed_candle_datetime
 
     # Check for incorrect configuration
-    if backtesting_from_ms >= backtesting_to_ms:
-        raise Exception("[ERROR] Backtesting periods are configured incorrectly.")
+    try:
+        timeframe = backtesting_from_ms - backtesting_to_ms
+        if timeframe > 0:
```

Comment:
```suggestion
        if timeframe >= 0:
```

If you don't include 0, then a nasty keyerror is thrown when the backtesting_from and backtesting_to dates are exactly the same